components/StoryCards/StoryCard.tsx

```diff
@@ -18,9 +18,10 @@ type Props = {
 function StoryCard({ story, size = 'small' }: Props) {
     const { categories, title, subtitle } = story;
     const { showDate, showSubtitle } = useThemeSettings();
-
+    const { isTablet } = useDevice();
+    const hasCategories = categories.length > 0;
     const HeadingTag = size === 'small' ? 'h3' : 'h2';
-
+    const shouldShowSubtitle = isTablet ? true : size !== 'small';
```

Comment:
this would show subtitles for small cards that are in tablet aspect ratio or lower, I think we should remove for all small cards?

Reply:
I guess we need to keep subtitles on mobile and tablet devices as their layout supports it, I'll ping Hugo and ask him for more details.